iModelCore/GeomLibs/PublicAPI/Geom/dellipse3d.h

```diff
@@ -113,7 +114,24 @@ double sweepRadians         //!< [in] sweep angle.
 //!
 static DEllipse3d FromPointsOnArc (DPoint3dCR start, DPoint3dCR middle, DPoint3dCR end);
 
-//! Return a circular arc with gven center and start.  Endpoint is on the vector to given endTarget.
+//!
+//! @description Create an elliptical arc from 3 points on ellipse: two points on an axis and one in between.
+//!
+//! @param [in] start start of arc, on an axis.
+//! @param [in] middle point on arc somewhere between `start` and `end`.
+//! @param [in] end point on arc directly opposite `start`.
+//! @param [in] sweep angular sweep (radian), measured from `start` in the direction of `middle`.
+//! @return The ellipse, or invalid carrier if construction impossible.
+//!
+static std::optional<DEllipse3d> FromStartMiddleEnd
+(
+DPoint3dCR start,
+DPoint3dCR middle,
+DPoint3dCR end,
+double sweep = msGeomConst_2pi
```

Comment:
I wanted to put `Angle::TwoPi()` instead of `msGeomConst_2pi` but could not resolve the compile error (could not find `Angle`)